data2.js

```diff
@@ -3,7 +3,7 @@ const data3 = [
     id: 1,
     speackerName: 'Yochai Benkler',
     speackerDesc: 'Yochai Benkler Lorem Ipsum is simply dd not only five cc typesetting, remaining essentonly five cc typesetting, remaining essent',
-    speackerImg: '../Images/pic3.jpg',
+    speackerImg: './Images/pic3.jpg',
```

Comment:
## ADD THE MISSING alt attribute on all IMAGES IN YOUR WEBPAGE

- Let's add the missing alt attribute and text to all the images in your webpage to improve the web accessibility of the page incase the image fails to load to aid the visually impaired people.